register_commands/register.js

```diff
@@ -8,12 +8,56 @@ const headers = {
   "Content-Type": "application/json"
 }
 
+let commands = []
+
 let command_data = {
   "name": "foo",
   "type": 1,
   "description": "replies with bar ;/",
 }
 
-axios.post(url, JSON.stringify(command_data), {
-  headers: headers,
-})
+commands.push(command_data)
+
+let command_data_with_options = {
```

Comment:
it should be parsed from `.json` file (`register_commands/exampleCommandWithOptions.json`). 

then we don't have to repeat it twice